### ISSUE TITLE
Version 5.3.0 acquires 3 video locks and never releases them, which prevents screen saver from kicking in

### ISSUE THREAD

**Issue by klentz75** [MehediH/Tweeten#224]
Running 5.3.0, the app continuously keeps 3 video locks, even when minimized.
The locks seem to never be released, and no video is playing anyway.

C:\Windows\system32>powercfg /requests
DISPLAY:
[PROCESS] \Device\HarddiskVolume3\Users\K\AppData\Local\Programs\Tweeten\Tweeten.exe
Video Wake Lock
[PROCESS] \Device\HarddiskVolume3\Users\K\AppData\Local\Programs\Tweeten\Tweeten.exe
Video Wake Lock
[PROCESS] \Device\HarddiskVolume3\Users\K\AppData\Local\Programs\Tweeten\Tweeten.exe
Video Wake Lock

**Comment by MehediH** [comment]
Disabling hardware acceleration seems to fix it.

**Comment by klentz75** [comment]
Disabling hardware acceleration only fixes it if there are no animated gifs showing anywhere on screen, and even then sometimes there are video wake locks for no apparent reason. I don't think animated gifs should acquire a video wake lock, unless maybe if the user opens them explicitly by clicking on them. Even then, the lock should be released on closing of the animated gif.
This was introduced in 5.3.0. Previous version also had this problem, but locks were acquired and never released only after clicking on an animated gif, whereas in the latest version it happens regardless.

**Comment by MehediH** [comment]
Duplicate of #231  and fixed in 5.4.0
